### ISSUE TITLE
Issue with pom-scijava 39

### ISSUE THREAD

**Issue by NicoKiaru** [bigdataviewer/bigvolumeviewer-core#21]
Hello!

I tried bvv and pom-scijava 39, currently I get this error:

```
Exception in thread "AWT-EventQueue-0" com.jogamp.opengl.GLException: Caught NoSuchMethodError: net.imglib2.view.Views.flatIterable(Lnet/imglib2/RandomAccessibleInterval;)Lnet/imglib2/IterableInterval; on thread AWT-EventQueue-0
	at com.jogamp.opengl.GLException.newGLException(GLException.java:76)
	at jogamp.opengl.GLDrawableHelper.invokeGLImpl(GLDrawableHelper.java:1327)
```

I hope it's an easy fix!

**Comment by NicoKiaru** [comment]
As @ctrueden [mentioned](https://imagesc.zulipchat.com/#narrow/stream/327237-SciJava/topic/pom-scijava.2039.20melting.20pot.20issues/near/476837727), one simply needs to recompile the repo with pom-scijava 39 (and imglib7).
Then it just works:

![image](https://github.com/user-attachments/assets/7d6ca3ef-6a8b-408c-a45f-b3fb5bcb2724)

**Comment by tpietzsch** [comment]
I updated the POM with the new dependency versions and released `bigvolumeviewer-0.3.4`
